tensorflow/core/kernels/conv_ops.cc

```diff
@@ -563,21 +563,6 @@ template struct LaunchConv2DOp<CPUDevice, float>;
 template struct LaunchConv2DOp<CPUDevice, double>;
 
 #if GOOGLE_CUDA || TENSORFLOW_USE_ROCM
-
-#if GOOGLE_CUDA
-<<<<<<< HEAD
-// Returns true if the given StreamExecutor is for a Volta or newer nvidia GPU.
```

Comment:
@parallelo we do want to keep this function. please refer to #29185

Reply:
Yes, agree.  But in the upstream repo, that function has been moved to `tensorflow/core/kernels/conv_ops_gpu.h`.  

Let me know if I'm still missing something.  

https://github.com/ROCmSoftwarePlatform/tensorflow-upstream/blob/develop-upstream-sync-190603/tensorflow/core/kernels/conv_ops_gpu.h#L30-L37

Reply:
Thanks for clarification. Please disregard my earlier comment then.

Reply:
Sure, no prob.  I appreciate your review of the changes.